content/test/integration/controller/active-entities.spec.ts

```diff
@@ -565,8 +568,9 @@ describe('Integration - Get Active Entities', () => {
 
       expect(response).toBeDefined()
       expect(response.length).toBe(1)
-      expect(response[0].entityId).toBe(secondDeploy.controllerEntity.id)
-      expect(response[0].pointer).toBe('urn:decentraland:mumbai:collections-thirdparty:athirdparty:wintercollection:1')
+      expect(response[0].pointers).toEqual(pointer.map((p) => p.toLocaleLowerCase()))
+      expect(response[0].id).toEqual(secondDeploy.controllerEntity.id)
```

Comment:
```
response[0].entityId => response[0].id
```

Watch out, breaking change

Reply:
Yeah, already checked with dApps team. Current endpoint was not used for a long time